projects/core/src/site-theme/providers/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './theme_storage_Id';
```

Comment:
In this PR sometimes we use term `theme` and sometimes `site theme`.

For consistency, let's rename everywhere in this pr the word `theme` to `site theme` (preserving the original casing, e.g.  theme` / `Theme` --> `siteTheme` OR  `site-theme` / `SiteTheme`

Reply:
done